src/validation.h

```diff
@@ -218,7 +218,7 @@ void UpdateCoins(const CTransaction& tx, CCoinsViewCache& inputs, int nHeight);
  *
  * See consensus/consensus.h for flag definitions.
  */
-bool CheckFinalTx(const CTransaction &tx, int flags = -1) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+bool CheckFinalTx(CBlockIndex* active_chain_tip, const CTransaction &tx, int flags = -1) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Comment:
This commit doesn't compile as `CheckFinalTx` is used L197  (`src/interfaces/chain.cpp`). But fixed in next commit AFAICT.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20158#discussion_r518464000

> This commit doesn't compile as `CheckFinalTx` is used L197 (`src/interfaces/chain.cpp`). But fixed in next commit AFAICT.

I'm seeing this problem in commit "validation: Pass in chain tip to ::CheckFinalTx" (5433a716e7f871204474117b0f9bdcd6ad8a16ce)